autogenerated.ts

```diff
@@ -37,6 +37,8 @@ namespace ml {
     return result;
   };
+
+  simulatorSendData();
```

Comment:
Also added this to the default `autogenerated.ts`, assuming it is needed, but not really tested it with the simulator itself.
Happy to remove it if it's better without it.

Reply:
We don't need this line. The `simulatorRegister()` call kicks things off. When the simulator is rendered it asks this extension for the data, so `simulatorSendData()` is only triggered in response.

Reply:
I think this line might still be generated in the `autogenerated.ts` file (or at least the files generated in the `ml4f-test-header` branch), should it be removed from there as well?

Reply:
Yes, good catch. Needs testing after the change, but I'm confident that's the right thing to do.